<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="serverStatuslabel">
      <property name="text">
       <string>正在初始化...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="QueryLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="QueryButton">
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Start"/>
   <addaction name="Stop"/>
   <addaction name="Log"/>
   <addaction name="freshaction"/>
  </widget>
  <action name="Start">
   <property name="text">
    <string>开启</string>
   </property>
  </action>
  <action name="freshaction">
   <property name="text">
    <string>刷新列表</string>
   </property>
  </action>
  <action name="Stop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="Log">
   <property name="text">
    <string>日志</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
